source/documentors/references/quick_reference_rst.rst

```diff
@@ -172,6 +221,111 @@ Use the following code:
 
 .. seealso:: To see alternative ways of defining tables, visit the `RST documentation about this topic <https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/style-guide/#tables>`_.
 
+Add a Substitution
```

Comment:
I think this section is more confusing than helpful. I've never used this - do we use it a lot?

Reply:
I have never used it either. I added it because it was on the previous [template page](https://docs.openedx.org/en/latest/documentors/references/doc_templates.html#add-a-substitution), and I wanted to include all the content in case it was necessary.
If you agree, I will erase it for the final version.